doc/source/ray-observability/ray-metrics.rst

```diff
@@ -41,30 +41,58 @@ First, download Prometheus. `Download Link <https://prometheus.io/download/>`_
     tar xvfz prometheus-*.tar.gz
     cd prometheus-*
 
-Let's modify Prometheus's config file to scrape metrics from Prometheus endpoints.
+Ray provides a prometheus config that works out of the box. After running ray, it can be found at `/tmp/ray/metrics/prometheus/prometheus.yml`.
 
 .. code-block:: yaml
 
-    # prometheus.yml
     global:
-      scrape_interval:     5s
-      evaluation_interval: 5s
+      scrape_interval: 15s
+      evaluation_interval: 15s
 
     scrape_configs:
-      - job_name: prometheus
-        static_configs:
-        - targets: ['localhost:8080'] # This must be same as metrics_export_port
+    # Scrape from each ray node as defined in the service_discovery.json provided by ray.
+    - job_name: 'ray'
+      file_sd_configs:
+      - files:
+        - '/tmp/ray/prom_metrics_service_discovery.json'
+
 
 Next, let's start Prometheus.
 
 .. code-block:: shell
 
-    ./prometheus --config.file=./prometheus.yml
+    ./prometheus --config.file=/tmp/ray/metrics/prometheus/prometheus.yml
 
 Now, you can access Ray metrics from the default Prometheus url, `http://localhost:9090`.
 
 See :ref:`here <multi-node-metrics>` for more information on how to set up Prometheus on a Ray Cluster.
 
+.. _grafana:
+
+Grafana
+-------
+(TODO: AGUO WIP [DO NOT MERGE])
+Grafana is a tool that supports more advanced visualizations of prometheus metrics and
+allows you to create custom dashboards with your favorite metrics. Ray exports some default
+configurations which includes a default dashboard showing some of the most valuable metrics
+for debugging ray applications.
+
+First, download Grafana. `Download Link <https://grafana.com/grafana/download>`
+
+Then run grafana using the built in configuration found in `/tmp/ray/metrics/grafana` folder.
+
+.. code-block:: shell
+
+    ./bin/grafana-server --config /tmp/ray/metrics/grafana/grafana.ini web
+
+Now, you can access grafana using the default grafana url, `http://localhost:3000`.
+If this is your first time, you can login with the username: `admin` and password `admin`.
+
+You can then see the default dashboard by going to dashboards -> manage -> Ray -> Default Dashboard.
+
+TODO(aguo): INSERT SCREENSHOT
+
```

Comment:
I think we also need a section for custom prometheus & Grafana config

Reply:
removed custom configs for now.